### PR TITLE
RC69 Fix for the point light test flashing in HMD

### PR DESCRIPTION
**Pull request by samcake**
Fixing the input format not being updated correctly in case of a stereo state change and instanced attributes

This is adressing this bug:
https://highfidelity.fogbugz.com/f/cases/15999/HMD-Rendering-Test-Error-rapid-flashing-colors-Point-Light

The same repo doesn't trigger the problem in this pr
Since the problem only happens in HMD/stereo, it s only worth testing on WIndows with HMD

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10817/HighFidelity-Beta-PR13440-48b0e2b.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10817/HighFidelity-Beta-Interface-PR13440-48b0e2b.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10817/HighFidelity-Beta-PR13440-48b0e2b.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10817/HighFidelity-Beta-Interface-PR13440-48b0e2b.exe

**Comment by qlimguicheteau** [comment]
Now Testing

**Comment by qlimguicheteau** [comment]
PASS @conklin94122 

Same computer that had the problem before now sees the area as a stable light.
Looks clean!

clean

File: libraries/gpu-gl-common/src/gpu/gl/GLBackendInput.cpp [hunk 1]
```diff
@@ -213,11 +221,12 @@ void GLBackend::updateInput() {
                     (void)CHECK_GL_ERROR();
                 }
 #ifdef GPU_STEREO_DRAWCALL_INSTANCED
-                glVertexBindingDivisor(bufferChannelNum, frequency * (isStereo() ? 2 : 1));
+                glVertexBindingDivisor(bufferChannelNum, frequency * (isStereoNow ? 2 : 1));
 #else
                 glVertexBindingDivisor(bufferChannelNum, frequency);
 #endif
             }
+            _input._lastUpdateStereoState = isStereoNow;
```
Comment (SamGondelman) [inline libraries/gpu-gl-common/src/gpu/gl/GLBackendInput.cpp]: setting this twice

**Comment by SamGondelman** [comment]
don't forget to make a master PR

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10822/HighFidelity-Beta-PR13440-67ff06f.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10822/HighFidelity-Beta-Interface-PR13440-67ff06f.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10822/HighFidelity-Beta-PR13440-67ff06f.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10822/HighFidelity-Beta-Interface-PR13440-67ff06f.exe

**Pull request merged**